authorizer-app/src/app/components/shared/toolbar/toolbar.component.html

```diff
@@ -1,9 +1,9 @@
 <mat-toolbar color="primary">
   <mat-toolbar-row>
     <div class="logo-wrapper">
-      <img class="logo" src="./assets/radar-logo.svg" alt="RADAR-base Logo" />
+      <img class="logo" src="./assets/chdr_logo.png" alt="CHDR Logo" />
```

Comment:
Please roll back these changes. This is the open-source RADAR-base version.

Reply:
Oh, ok. I thought we want to change the repo!